src/flyte/_debug/constants.py

```diff
@@ -12,7 +12,6 @@
 }
 DEFAULT_CODE_SERVER_EXTENSIONS = [
     "https://raw.githubusercontent.com/flyteorg/flytetools/master/flytekitplugins/flyin/ms-python.python-2023.20.0.vsix",
-    "https://raw.githubusercontent.com/flyteorg/flytetools/master/flytekitplugins/flyin/ms-toolsai.jupyter-2023.9.100.vsix",
```

Comment:
Downloading and installing this extension takes 10+ seconds, and we usually don’t use it, so I removed it here